### PR TITLE
refactor: save method

### PR DESCRIPTION
**Pull request by ashikka**

### CODE REVIEW

File: src/containerTest/kotlin/com/quipalup/katydid/DeleteLogEntryByIdCT.kt [hunk 1]
```diff
@@ -37,7 +37,7 @@ class DeleteLogEntryByIdCT {
 
     @Test
     fun `delete long entry by id`() {
```
Comment (cristinaverdi) [inline src/containerTest/kotlin/com/quipalup/katydid/DeleteLogEntryByIdCT.kt:39]: long -> log

File: src/main/kotlin/com/quipalup/katydid/logentry/secondaryadapter/database/LogEntryDatabase.kt [hunk 2]
```diff
@@ -32,26 +31,25 @@ class LogEntryDatabase(private val jpaLogEntryRepository: JpaLogEntryRepository)
...
+    private fun JpaLogEntry.ensureDoesNotExist(): Either<SaveLogEntryError, JpaLogEntry> =
         jpaLogEntryRepository.existsById(this.id)
             .let {
                 when (it) {
-                    true -> CreateLogEntryError.AlreadyExists.left()
+                    true -> SaveLogEntryError.AlreadyExists.left()
                     false -> this.right()
                 }
             }
+    private fun JpaLogEntry.ensureEntityExists(): Either<SaveLogEntryError, JpaLogEntry> =
```
Comment (cristinaverdi) [inline src/main/kotlin/com/quipalup/katydid/logentry/secondaryadapter/database/LogEntryDatabase.kt:45]: yeah!

File: src/main/kotlin/com/quipalup/katydid/logentry/secondaryadapter/database/LogEntryDatabase.kt [hunk 3]
```diff
@@ -17,8 +16,8 @@ import org.springframework.data.repository.findByIdOrNull
 
 @Named
 class LogEntryDatabase(private val jpaLogEntryRepository: JpaLogEntryRepository) : LogEntryRepository {
-    override fun create(logEntry: LogEntry): Either<CreateLogEntryError, Id> = logEntry.toJpa()
-        .flatMap { jpaLogEntry: JpaLogEntry -> jpaLogEntry.ensureDoesNotExist() }
+    override fun save(logEntry: LogEntry): Either<SaveLogEntryError, Id> = logEntry.toJpa()
+//        .flatMap { jpaLogEntry: JpaLogEntry -> jpaLogEntry.ensureDoesNotExist() }
```
Comment (cristinaverdi) [inline src/main/kotlin/com/quipalup/katydid/logentry/secondaryadapter/database/LogEntryDatabase.kt:20]: I guess you will uncomment this line over here?

Comment (ashikka) [reply src/main/kotlin/com/quipalup/katydid/logentry/secondaryadapter/database/LogEntryDatabase.kt:20]: Yes, I have to still change things around here

Comment (cristinaverdi) [reply src/main/kotlin/com/quipalup/katydid/logentry/secondaryadapter/database/LogEntryDatabase.kt:20]: Hello, I reviewed again your PR, this line is not necessary in this layer. 

Let's move that logic to the application layer. Please see the comment below

**Review by cristinaverdi — APPROVED** [review]

**Pull request merged**